pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -1306,13 +1306,34 @@ func testHTTPSClassification(t *testing.T, tr *Tracer, clientHost, targetHost, s
 				require.NoError(t, err)
 				t.Cleanup(closer)
 			},
-			postTracerSetup: func(t *testing.T, ctx testContext) {
```

Comment:
postTracer is mandatory to have (ATM)
so all of your tests have panic

Reply:
👍 I'll add a `nil` check for the callback functions